content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md

```diff
@@ -527,9 +527,7 @@ profiles:
 ```
 
 {{< note >}}
-The [`SelectorSpread` plugin](/docs/reference/scheduling/config/#scheduling-plugins)
-is disabled by default. The Kubernetes project recommends using `PodTopologySpread`
-to achieve similar behavior.
+The 'PodTopologySpread' plugin is enabled by default. 
```

Comment:
I think we can remove this. It's well known.
/lgtm cancel

Reply:
@tamilselvan1102 Still working on this?

Reply:
Updated as per @kerthcet  comment and waiting for approval

Reply:
@tamilselvan1102 Forgot to `git commit --amend` or `git push -f`?

Reply:
Changes are reflecting in the preview page 
https://deploy-preview-42759--kubernetes-io-main-staging.netlify.app/docs/concepts/scheduling-eviction/topology-spread-constraints/#cluster-level-default-constraints

Reply:
@tamilselvan1102 The suggestion was to remove the note completely, and it seemed that you agreed to this suggestion but you didn't "updated as per @kerthcet comment".

Reply:
I was misunderstood. now removed. Please confirm the changes